### PR TITLE
Improve doc of collective.js.bootstrap pin.

### PR DESCRIPTION
**Pull request by idgserpro**
[ci skip]

### CODE REVIEW

File: versions-4.3.x.cfg [hunk 1]
```diff
@@ -14,6 +14,7 @@ test-eggs =
     plone.app.referenceablebehavior
 
 [versions]
+# https://github.com/collective/collective.cover/issues/492
```
Comment (hvelarde) [inline versions-4.3.x.cfg]: add `XXX: ` in front to keep consistency.

Comment (idgserpro) [inline versions-4.3.x.cfg]: Done

**Comment by idgserpro** [comment]
I think we have a problem in github's workflow for accepting a PR when a `[ci skip]` is added to a commit. @mauritsvanrees have you ever seen something like this?

**Comment by idgserpro** [comment]
Just out of curiosity @hvelarde, is there a controlpanel where you can "retreat" the pre-requisites of a PR approval?

**Comment by hvelarde** [comment]
as an administrator I can skip those requirements :sunglasses:

**Pull request merged**